src/main/java/eu/yals/constants/App.java

```diff
@@ -14,7 +14,9 @@ private App() {
     public static final String AND = "&";
     public static final String AT = "@";
     public static final String NO_VALUE = "_";
+    public static final int NO_STATUS = -1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)